src/pages/about.astro

```diff
@@ -170,17 +157,56 @@ import "animate.css";
 					</p>
 				</div>
 			</div>
+
+			<a href='/careers' class="p-2"><p>Want to work with us? See what openings we have available ⤳</p></a>
+
 		</div>
-		<div id='our-cooperative' class='mt-16'>
+		<div id='our-cooperative'>
 			<h2 class='text-[#232656] font-bold' style='line-break: anywhere;'>
 				How we run our cooperative.
 			</h2>
 			<p class='text-[#232656]'>
-				Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-				tempor incididunt ut labore et dolore magnam aliquam quaerat voluptatem.
-				Ut enim aeque doleamus animo, cum corpore dolemus, fieri tamen permagna
-				accessio potest, si aliquod aeternum et infinitum impendere malum nobis
-				opinemur. Quod idem licet transferre.
+				We believe in the power of cooperatives to fundamentally change the way we live and work, and With the Ranks was built to provide a more sustainable model for building organizing tech. Here are a few core beliefs we have about how our business should operate:
+			</p>
+			<h3 class='text-[#232656] font-bold text-3xl' style='line-break: anywhere;'>
```

Comment:
Just carrying over the note about this styling (also in a few other places) from Slack!
```suggestion
			<h3 class='text-[#232656] font-bold text-3xl'>
```